### PR TITLE
Community friendly rework

### PR DESCRIPTION
**Pull request by vferat**
- [x] Add Contributing
- [x] Add Citing
- [x] Add install

### CODE REVIEW

File: README.md
Comment (mscheltienne) [inline README.md:2]:
```suggestion
[![PyPI version](https://badge.fury.io/py/pycrostates.svg)](https://badge.fury.io/py/pycrostates)
[![Conda Version](https://img.shields.io/conda/vn/conda-forge/pycrostates.svg)](https://anaconda.org/conda-forge/pycrostates)
```

File: README.md
Comment (mscheltienne) [inline README.md:3]:
```suggestion
```

Duplicated of badge below?

File: README.md
Comment (mscheltienne) [inline README.md:20]: I am not getting the same numbers, and the link is only to the image?

Suggestion for Pypi:
`[Downloads](https://pepy.tech/project/pycrostates)`

Suggestion for conda:
`[Anaconda-Server Badge](https://anaconda.org/conda-forge/pycrostates)`

I don't find a download / month for `conda`.

File: README.md
Comment (vferat) [reply README.md:4]: I don't think this is not really an issue. Feel free to delete it  before merging if you think it's better

**Comment by mscheltienne** [comment]
DOI is not rendering anymore:

Screenshot 2022-08-10 at 15 59 00

**Review by mscheltienne — APPROVED** [review]
LGTM, merge if happy. The render of the DOI badge fixed itself.

**Pull request merged**
